utils/update_translation_stats.py

```diff
@@ -100,35 +98,33 @@ def generate_translation_stats(po_dir, output_file):
             cells = line.split(',')
             po_filename = cells[0]
             if po_filename.endswith('.po'):
-                entry = TranslationStats()
                 locale = regex_po.match(po_filename).group(1)
-                if locale in locale_stats:
-                    entry = locale_stats[locale]
-                entry.total = entry.total + int(cells[total_column])
-                entry.translated = entry.translated + \
-                    int(cells[translated_column])
+                entry = locale_stats[locale]
+                entry.total += int(cells[total_column])
+                entry.translated += int(cells[translated_column])
                 if entry.translated > entry.total:
-                    print('Error! Translated ' + str(entry.translated) + ' (+' + (cells[
-                          translated_column]) + ') is bigger than the total of ' + str(entry.total) + '(' + (cells[total_column]) + ')\n' + line)
+                    print('Error! Translated ' + str(entry.translated) + ' (+' +
+                          cells[translated_column] + ') is bigger than the total of ' +
+                          str(entry.total) + '(' + cells[total_column] + ')\n' + line)
                     sys.exit(1)
                 locale_stats[locale] = entry
 
     print('\n\nLocale\tTotal\tTranslated')
     print('------\t-----\t----------')
 
     # The total goes in a [global] section and is identical for all locales
-    result = '[global]\n'
-    result = '{}total={}\n\n'.format(
+    result = '# This file is managed by utils/update_translations_stats.py\n\n[global]\n'
+    result = '{}total={}\n'.format(
```

Comment:
I think `result += 'total={}\n'` could be used instead of this…

Reply:
It was similar to this before 067d75e4015 (update from python2 to python3), but `+=` still works, no idea why it was changed. (There is a 3rd occurrence of this.)